### DISCUSSION TITLE
Add charts

### DISCUSSION THREAD

**Discussion by jmrplens** [alshedivat/al-folio#457] [Ideas]
I have seen [this template](https://github.com/kitian616/jekyll-TeXt-theme) with [chart.js](https://github.com/chartjs/Chart.js), I find this type of graph interesting for a template oriented to research.
If I can I will try to implement it and make a pull request, but at the moment I don't have time.

[<img width="613" alt="Screenshot of Google Chrome (16-11-21, 12-51-30)" src="https://user-images.githubusercontent.com/28966312/141980657-8ab9c7fe-04d1-4b13-89ab-a93508e5a263.png">](https://www.chartjs.org/docs/latest/charts/line.html)

[<img width="615" alt="Screenshot of Google Chrome (16-11-21, 12-51-41)" src="https://user-images.githubusercontent.com/28966312/141980782-3fd0159e-af09-49bc-89aa-bed30b0aad4c.png">](https://www.chartjs.org/docs/latest/charts/bar.html)

[<img width="485" alt="Screenshot of Google Chrome (16-11-21, 12-52-10)" src="https://user-images.githubusercontent.com/28966312/141980853-4ee692b9-1769-4b9a-8e61-17fa898bfe98.png">](https://www.chartjs.org/docs/latest/charts/polar.html)

**Comment by alshedivat** [comment]
thanks for the suggestion! this is a good idea. I created a feature request issue to keep track of things.
